start.py

```diff
@@ -402,7 +402,8 @@ def cm_server_not_dead(primary_node):
         _configure_after_start(deployment=deployment,
                                cluster_name=DEFAULT_CLUSTER_NAME,
                                cluster=cluster, quiet=not args.verbose,
-                               kerberos_principals=args.kerberos_principals)
+                               kerberos_principals=args.kerberos_principals,
+                               cdh_version_tuple=cdh_version_tuple)
```

Comment:
I think this is something we can figure out from the Deployment object and the cluster name, no? Having to pass it ourselves seems like it could be redundant.